src/test/java/io/vertx/test/core/RedeploymentTest.java

```diff
@@ -88,7 +88,13 @@ public void setUp() throws Exception {
 
   @Override
   public void tearDown() throws Exception {
-    vertx.fileSystem().deleteRecursiveBlocking(tempDir.getPath(), true);
+    /* One of the temp files is a jar which is added to the classpath to test verticle redeploy on jar
+     * changes.  This jar remains locked by the JVM until all references to this class loader have been
+     * removed and garbage collected.  The following clean up statement fails because either there is
```

Comment:
The problem with TemporaryFolder is that it fails silently.  I think we want to see failed tests if something changes that prevents the folder from being deleted (on Linux / in CI.)